spyql/processor.py

```diff
@@ -129,6 +129,27 @@ def handle_1st_data_row(self, row):
         # dict of {col1: value1, ...}
         self.translations["row"] = f"NullSafeDict(zip(_names, {cols_expr}))"
 
+    def expand_col_names(self, all_col_names, name):
+        """
+        Expands column wildcard operator and modifiers
+        """
+        except_pattern = re.compile(r"\*\s*EXCEPT\s*\((.*)\)", re.IGNORECASE)
+        except_match = re.search(except_pattern, name)
+
+        if name == "*":
+            return all_col_names
+        elif except_match:
+            except_col_names = [
+                col_name.strip() for col_name in except_match.group(1).split(",")
+            ]
+            return [
+                col_name
+                for col_name in all_col_names
+                if col_name not in except_col_names
```

Comment:
Should we test if there are  columns that are in `except_col_names ` but not in `all_col_names` and throw a warning like "EXCEPT column {x} not found"?